apps/angular/src/app/features/anime-table/anime-table/anime-table.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component } from '@angular/core';
+import { AnimeService } from '@js-camp/angular/core/services/anime.service';
+import { AnimeItem } from '@js-camp/core/models/animeItem';
+import { Observable } from 'rxjs';
+
+/** Anime table component. */
+@Component({
+	selector: 'anime-table',
+	templateUrl: './anime-table.component.html',
+	styleUrls: ['./anime-table.component.css'],
+})
+export class AnimeTableComponent {
+	/** Array of anime items. */
+	protected readonly animeList$: Observable<AnimeItem[]>;
+
+	/** Columns to be displayed in the table. */
+	public readonly displayedColumns: string[] = ['titleEng', 'image', 'titleJpn', 'airedStartDate', 'type', 'status'];
```

Comment:
I guess you want a static array, not dynamic
```suggestion
	public readonly displayedColumns: readonly string[] = ['titleEng', 'image', 'titleJpn', 'airedStartDate', 'type', 'status'];
```